Fatras/Geant4/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+add_library(
+  ActsFatrasGeant4 SHARED
+  src/Geant4Decay.cpp
+  src/PDGtoG4Converter.cpp
+  src/DummyDetectorConstruction.cpp)
+target_compile_definitions(
+  ActsFatrasGeant4
+  PUBLIC ${Geant4_DEFINITIONS})
+target_include_directories(
+  ActsFatrasGeant4
+  SYSTEM PUBLIC ${Geant4_INCLUDE_DIRS})
+target_compile_features(
+  ActsFatrasGeant4
+  PUBLIC cxx_std_17)
+target_include_directories(
+  ActsFatrasGeant4
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>)
+target_link_libraries(
+  ActsFatrasGeant4
+  PUBLIC ActsCore ActsFatras ${Geant4_LIBRARIES})
+
+install(
+  TARGETS ActsFatrasGeant4
+  EXPORT ActsFatrasTargets
```

```suggestion
  EXPORT ActsFatrasGeant4Targets
```
This has to match the component name.

---

Done